Source/WebInspectorUI/UserInterface/Views/SettingsTabContentView.js

```diff
@@ -493,6 +493,11 @@ WI.SettingsTabContentView = class SettingsTabContentView extends WI.TabContentVi
 
         let engineeringSettingsView = new WI.SettingsView("engineering", WI.unlocalizedString("Engineering"));
 
+        if (WI.isEngineeringBuild) {
+            engineeringSettingsView.addSetting(WI.unlocalizedString("Debug UI:"), WI.showDebugUISetting, WI.unlocalizedString("Show Debug UI"));
+            engineeringSettingsView.addSeparator();
+        }
```

Comment:
i would've moved this to be the last item in the Engineering pane since most other WebKit developers will not want to use this feature (i.e. put it out of the way for everyone except for Web Inspector developers)